ansible/roles/csi-cephfs-fyre/files/csi-ceph.sh

```diff
@@ -7,6 +7,16 @@ oc login -u kubeadmin -p "$(cat /root/auth/kubeadmin-password)" https://api.$(ho
 rm -rf rook
 echo "Doing clone of rook release $rookRelease"
 git clone https://github.com/rook/rook.git -b $rookRelease
+# if rook-ceph is version 1.5, then need to create/apply crd
+majorRelease=$(echo ${rookRelease:0:4})
+if [[ $majorRelease != "v1.4" ]]
```

Comment:
Made the if condition like so because I am assuming that moving forward, no one will use a rook ceph version earlier than 1.4